.github/workflows/update_hub.yml

```diff
@@ -51,6 +51,16 @@ jobs:
           git add .
           git commit -m "Update MOC files" -a  || echo "nothing to commit"
 
+      # This must be the last edit, to ensure that no later steps accidentally
+      # overwrite the footer. 
+      - name: Add Footer
+        run: |
+          cd .github/scripts
+          python3 ./add_footer.py
+          cd ../..
+          git add .
+          git commit -m "Add 'This note in GitHub' footers" -a  || echo "nothing to commit"
```

Comment:
Does it need the `-a` flag?
I think it shouldn't be necessary because of the `git add` before it.
https://git-scm.com/docs/git-commit

Reply:
Thank you... I've removed the `-a` in this new section, and, for consistency, in the earlier steps that had their own `git add` line.

(The difference is that some steps potentially add new files, whereas the first one only ever updates existing files)

Reply:
Would appreciate a review of 0f3ba7e @kometenstaub - no hurry...

Reply:
Thank you for the explanation and for making it consistent. :)